src/main/java/com/faforever/client/preferences/event/MissingGamePathEvent.java

```diff
@@ -1,5 +1,14 @@
 package com.faforever.client.preferences.event;
 
-public enum MissingGamePathEvent {
-  INSTANCE
+import lombok.AllArgsConstructor;
+import lombok.Value;
+
+@Value
```

Comment:
Oh boy, I see all the classes passing in front of me where I should've used this instead of @Data :)